diff --git a/config/app_config.yml.sample b/config/app_config.yml.sample
index f204a45c91..fa946204cb 100644
--- a/config/app_config.yml.sample
+++ b/config/app_config.yml.sample
@@ -10,11 +10,11 @@ defaults: &defaults
     memory_limit:
   debug_assets: true
   mandatory_keys:     [layer_opts, sql_api, varnish_management, redis, session_domain]
-  session_domain:     '.localhost.lan'
+  session_domain:     'localhost'
   # If activated, urls will use usernames in format //SESSION_DOMAIN/user/USERNAME and ignore subdomains if present
-  subdomainless_urls: false
+  subdomainless_urls: true
   http_port:           3000 # nil|integer. HTTP port to use when building urls. Leave empty to use default (80)
-  https_port:          # nil|integer. HTTPS port to use when building urls. Leave empty to use default (443)
+  https_port:          3000 # nil|integer. HTTPS port to use when building urls. Leave empty to use default (443)
   # Setting this to true will enable ActiveController's enforcement of SSL.
   ssl_required: false
   secret_token:       '71c2b25921b84a1cb21c71503ab8fb23'
@@ -22,14 +22,14 @@ defaults: &defaults
   secret_key_base:    '65903fa751affcdd71a9eb09308bcb404c50c8df03414db849ea22fbe8d4aae9ff344f6594630eb9c8367b4fd8ed2211d0342a49df473dccc27ae0be120b25ab'
   # It's recommended to generate a new password_secret (pepper for password encryption) for each installation using `bundle exec rake secret`
   password_secret:    'cc914bbbfa1d7156ec1424260d3a3214f3d1720aae872c1a5887f918929d0fec6f048d708f1ca79b4e43d20b4b623726e656917c9b8f2de0ea6c18e2ee80ce89'
-  account_host:       'localhost.lan:3000'
+  account_host:       'localhost:3000'
   # Here you can define other hosts different to account_host that also will be CORS enabled
   # cors_enabled_hosts:
   #   - example.com
   #   - foo.bar
   #   - carto.dev
   account_path:       '/account'
-  vizjson_cache_domains: ['.localhost.lan']
+  vizjson_cache_domains: ['localhost']
   data_library:
     username:         'common-data'
     path:             '/data-library'
@@ -40,33 +40,33 @@ defaults: &defaults
     filter: 'mapnik'
     internal:
       protocol:      'http'
-      domain:        'localhost.lan'
+      domain:        'localhost'
       port:          '8181'
       host:          '127.0.0.1'
       verifycert:     false
     private:
       protocol:      'http'
-      domain:        'localhost.lan'
+      domain:        'localhost'
       port:          '8181'
       verifycert:     false
     public:
       protocol:      'http'
-      domain:        'localhost.lan'
+      domain:        'localhost'
       port:          '8181'
       verifycert:     false
   sql_api:
     private:
       protocol:   'http'
-      domain:     'localhost.lan'
+      domain:     'localhost'
       endpoint:   '/api/v1/sql'
       port:       8080
     public:
       protocol:   'http'
-      domain:     'localhost.lan'
+      domain:     'localhost'
       endpoint:   '/api/v2/sql'
       port:       8080
   api_requests_service_url: ''
-  developers_host:    'http://developers.localhost.lan:3000'
+  developers_host:    'http://developers.localhost:3000'
   google_analytics:
     primary:          ''
     embeds:           ''
@@ -209,7 +209,7 @@ defaults: &defaults
     # secondary:
     #   host: '127.0.0.1'
   org_metadata_api:
-    host: 'localhost.lan'
+    host: 'localhost'
     port: '3000'
     username: "extension"
     password: "elephant"
@@ -320,10 +320,10 @@ defaults: &defaults
         interactivity:      "cartodb_id"
         debug:              false
         visible:            true
-        tiler_domain:       "localhost.lan"
+        tiler_domain:       "localhost"
         tiler_port:         "80"
         tiler_protocol:     "http"
-        sql_domain:         "localhost.lan"
+        sql_domain:         "localhost"
         sql_port:           "80"
         sql_protocol:       "http"
         extra_params:       { cache_policy: 'persist' }
@@ -356,7 +356,7 @@ defaults: &defaults
     #   max_size_in_bytes: 1048576 # Optional, default is 1 MB
     #   location: 'html_assets' # Optional subdirectory for local assets, default is 'html_assets'
   app_assets:
-    asset_host: "//cartodb-libs.global.ssl.fastly.net/cartodbui"
+    # asset_host: "//cartodb-libs.global.ssl.fastly.net/cartodbui"
   avatars:
     gravatar_enabled: true
     base_url: '/assets/unversioned/images/avatars'
@@ -838,9 +838,9 @@ test:
     port: 6335
   enforce_non_empty_layer_css: false
   api_requests_es_service:
-    url: "http://api-calls-service.localhost.lan/search"
+    url: "http://api-calls-service.localhost/search"
     body: ""
-  session_domain:     '.localhost.lan'
+  session_domain:     'localhost'
   subdomainless_urls: false
   federated_server:
     dir: "/tmp/federated_server_db"
diff --git a/config/database.yml.sample b/config/database.yml.sample
index 9faa50fabd..198451df36 100644
--- a/config/database.yml.sample
+++ b/config/database.yml.sample
@@ -34,7 +34,7 @@ development:
   username: postgres
   password:
   conn_validator_timeout: 900
-  connect_timeout: 3
+  connect_timeout: 10
   pool: 50
   prepared_statements: false
 
