diff --git a/script/resque b/script/resque
index 57aa704313..598d76949a 100755
--- a/script/resque
+++ b/script/resque
@@ -1,2 +1,2 @@
 #!/bin/sh
-VVERBOSE=true QUEUE=imports,exports,users,user_dbs,geocodings,synchronizations,tracker,user_migrations,batch_updates,gears rake environment resque:work
+QUEUE=imports,exports,users,user_dbs,geocodings,synchronizations,tracker,user_migrations,batch_updates,gears rake environment resque:work
